pebblo/reports/templates/weasyprintTemplate.html

```diff
@@ -140,9 +142,11 @@
             </table>
           </div>
         </div>
+        {% if data.loadHistory.history|length > 5 %}
```

Comment:
We are sending 5 entries from backend. So we need to have check as length >= 5.

Reply:
Updated check